scrapper.py

```diff
@@ -25,39 +39,159 @@ class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls, max_articles: int):
-        pass
+
+    def __init__(self, seed_urls, total_max_articles: int):
+        self.seed_urls = seed_urls
+        self.total_max_articles = total_max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        article_summaries_bs = article_bs.find_all("div", class_="obj_article_summary")
+        for article_summary_bs in article_summaries_bs:
+            link_to_pdf = article_summary_bs.find('a', class_='obj_galley_link pdf')
+            if link_to_pdf:
```

Comment:
remember: you need to make your code shallow. that will be one of your tasks next Friday if you do not make it earlier.

Reply:
Done